README.md

```diff
@@ -216,10 +216,16 @@ The custom configuration options for the Celery workers are listed below:
 In addition to building operator index images, IIB can also be used to regenerate operator bundle
 images. This is useful for applying modifications to the manifests embedded in the bundle image.
 IIB uses the [operator-manifest](https://github.com/containerbuildsystem/operator-manifest) library
-to assist in these modifications. Currently, IIB will pin any container image pull specification
-to its corresponding digest. See the different
+to assist in these modifications.
+
+Currently, IIB will pin any container image pull specification
+to its corresponding digest if
```

Comment:
can we please reword this sentence. It caused a bit of confusion when read together with `IIB also sets
[spec.relatedImages](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/operators/index#olm-enabling-operator-for-restricted-network_osdk-generating-csvs)
when it isn't already set.`

Reply:
Or did you mean to end this statement with `is not set` => `is set`?

Reply:
Well IIB only pins the pull specifications if `spec.relatedImages` isn't set and it only sets `spec.relatedImages` when it's not already set. I'll make this clearer.